R/urls.R

```diff
@@ -1,5 +1,12 @@
 
 r_svn_url <- "https://svn.r-project.org/R/tags/"
-r_download_url <- "https://cran.r-project.org/src/base/R-3/"
-r_win_download_url <- "https://cran.r-project.org/bin/windows/base/"
-r_macos_download_url <- "https://cran.r-project.org/bin/macosx/"
+
+if (nzchar(Sys.getenv("R_RVERSIONS_CRAN_URL"))){
+  cran_url <- Sys.getenv("R_RVERSIONS_CRAN_URL")
+} else {
+  cran_url <- "cran.r-project.org"
+}
```

Comment:
This is running at install time, which is not what we want I guess. We'll need to put these into a function.